### PR TITLE
add Oldje.yml

### PR DESCRIPTION
**Pull request by JaseNZC**
A redo of this, Tested with old and new scenes on both sites and seems to work ok.

Date is present on one site but not the other.

In theory should be able to close the other PR of this site.

### CODE REVIEW

File: scrapers/Oldje.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - oldje-3some.com/videos
+    scraper: oldje3someScraper
+xPathScrapers:
+  oldjeScraper:
+    scene:
+      Studio:
+         Name:
+           fixed: Oldje
+      Title:
+        selector: //h1[@class='inlay_title']
```
Comment (bnkai) [inline scrapers/Oldje.yml:18]: the older scenes seem to have a different layout eg `https://www.oldje.com/video/lolita-taylor/dirty-santa`
```yaml
        selector: //h1[@class='inlay_title']|//h1[@class="preview_header"]/span[@class="bold"]
```

File: scrapers/Oldje.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          selector: //a[contains(@href,'tag/')]
+      Date:
+        selector: //span[@id="updata"]/text()
+        postProcess:
+          - parseDate: 2006-01-02
+      Performers:
+        Name:
+          selector: //a[@class="act_name_h"]/text()
+      Details:
+        selector: //p[@class='text']
```
Comment (bnkai) [inline scrapers/Oldje.yml:30]:
```yaml
        selector: //p[@class='text']|//div[@class="preview_desc"]
```
different for older scenes

File: scrapers/Oldje.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      Image:
+         selector: //div[@id="content"]/a/img/@src
```
Comment (bnkai) [inline scrapers/Oldje.yml:32]: same here
```yaml
         selector: (//div[@id="content" or @id="prev_m"]/a)[1]/img/@src
```

File: scrapers/Oldje.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  oldje3someScraper:
+    scene:
+      Studio:
+         Name:
+           fixed: Oldje-3some
+      Title:
+        selector: //h2[@class="page-title"]
```
Comment (bnkai) [inline scrapers/Oldje.yml:39]: from the scenes i tried this didnt seem to match
```yaml
        selector: //h1
```
seemed to work ok

**Comment by JaseNZC** [comment]
Yay should be ok now.

File: scrapers/Oldje.yml [hunk 5]
```diff
@@ -15,7 +15,7 @@ xPathScrapers:
          Name:
            fixed: Oldje
       Title:
-        selector: //h1[@class='inlay_title']
+         selector: //h1
```
Comment (bnkai) [inline scrapers/Oldje.yml:18]: this is not why we want
In old scenes `https://www.oldje.com/video/lolita-taylor/dirty-santa` h1 includes the performers so the title you get is not valid. (has performers in it)

**Comment by JaseNZC** [comment]
Bugger that was my mistake I don't know how I missed that. Should cover old and new scenes now.

**Pull request merged**